src/lnn.jl

```diff
@@ -0,0 +1,42 @@
+"""
+Constructs a Lagrangian Neural Network [1].
+
+References:
+[1] Miles Cranmer, Sam Greydanus, Stephan Hoyer, Peter Battaglia, David Spergel, and Shirley Ho.Lagrangian Neural Networks.
+    InICLR 2020 Workshop on Integration of Deep Neural Modelsand Differential Equations, 2020.
+"""
+
+struct LagrangianNN
+    model
+    re
+    params
```

Comment:
Just for consistency with the other models

```suggestion
    p
```